roles/core/dns_client/templates/resolv.conf.j2

```diff
@@ -1,7 +1,7 @@
 #jinja2: lstrip_blocks: True
 #### Blue Banquise file ####
 ## {{ ansible_managed }}
-{% set nameservers = networks[j2_node_main_network]['services_ip']['dns_ip'] | ipaddr %}
+{% set nameservers = networks[j2_node_main_network]['services_ip']['dns_ip'] %}
```

Comment:
Why do you remove the ipaddr filter?

Reply:
Because it forces to install an rpm on client nodes. I try to avoid having any non vital rpms installed on client nodes.
System administrator should see the error when using dns_server role.

Reply:
Are you sure about this? I will double check, but as far as I know this is used during templating on the ansible controller, not on the client side.

Reply:
See above, I agree with you